### ISSUE TITLE
add badges in NostrGit

### ISSUE THREAD

**Issue by rusteroobs** [NostrGit/NostrGit#220]
Hello, 

I would like to know if you all already think about adding badges to nostr. Have you all thought about adding badges to NostrGit? For example, something like: [badges/shields](https://github.com/badges/shields)

`https://img.nostrgit.io/badge/<LABEL>-<MESSAGE>-<COLOR>`

**Comment by cypherhoodlum** [comment]
Yes, we should add badges to the profile page and hover cards etc. Thanks for creating the issue :+1:

**Comment by marcelolucario** [comment]
> Yes, we should add badges to the profile page and hover cards etc. Thanks for creating the issue 👍


thanks!
